staging/knative/values.yaml

```diff
@@ -1,4 +1,19 @@
 global:
   serviceLabels: {}
+
+eventing:
+  enabled: true
+
+eventing-sources:
+  enabled: true
+
 serving:
+  enabled: true
   domain: example.com
+  customMetricsApiservice:
+    enabled: true
+  gateway:
+    https:
+      enabled: true
+  namespace:
```

Comment:
Could we get a more descriptive name for both the resource file and this field. To what namespace will these additional labels be created for?

Reply:
https://github.com/mesosphere/charts/pull/524#discussion_r399561876